tests/test_pyglmnet.py

```diff
@@ -44,28 +44,38 @@ def test_tikhonov():
     Tau = 1. / np.sqrt(np.float(n_samples)) * Tau / Tau.max()
 
     # fit model with batch gradient
-    glm_tikhonov = GLM(distr='poisson', alpha=0.0, Tau=Tau, solver='batch-gradient', tol=1e-5)
+    glm_tikhonov = GLM(distr='softplus',
+                       alpha=0.0,
+                       Tau=Tau,
+                       solver='batch-gradient',
+                       tol=1e-5,
+                       score_metric='pseudo_R2')
     glm_tikhonov.fit(Xtrain, ytrain);
 
     ytrain_hat = glm_tikhonov[-1].predict(Xtrain)
     ytest_hat = glm_tikhonov[-1].predict(Xtest)
 
     R2_train = dict()
     R2_test = dict()
-    R2_train['tikhonov'] = glm_tikhonov[-1].score(ytrain, ytrain_hat, np.mean(ytrain), method='pseudo_R2')
-    R2_test['tikhonov'] = glm_tikhonov[-1].score(ytest, ytest_hat, np.mean(ytrain), method='pseudo_R2')
+    R2_train['tikhonov'] = glm_tikhonov[-1].score(Xtrain, ytrain)
+    R2_test['tikhonov'] = glm_tikhonov[-1].score(Xtest, ytest)
 
     # fit model with cdfast
-    glm_tikhonov = GLM(distr='poisson', alpha=0.0, Tau=Tau, solver='cdfast', tol=1e-5)
+    glm_tikhonov = GLM(distr='softplus',
+                       alpha=0.0,
+                       Tau=Tau,
```

Comment:
This was part of PR #150 and `Tau` is a matrix, so I chose to use upper case. We have only `flake8` in CI.